src/utilities/index.ts

```diff
@@ -1,29 +1,54 @@
 import Link from 'gatsby-link'
 
-export function checkBreakpoint(breakpoint) {
+export function checkBreakpoint(breakpoint: number) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'checkBreakpoint' to have a typedef](https://app.codacy.com/app/PrideInLondon/pride-london-web/pullRequest?prid=4553972)